### PR TITLE
Adding .sub_labels

### PR DESCRIPTION
**Pull request by jrubenoff**
Fixes dobtco/screendoor-v2#1228

### CODE REVIEW

**Comment by ajb** [comment]
This looks great in the screenshots you've provided, but without `dvl-core` it looks a little screwy. Remember that `formrenderer-base` does _not_ depend on dvl-core, and even though there are barely any visual styles in this repo, the layout should still render properly. (This is important for embeds.)

**Comment by ajb** [comment]
Check out the updated version, lmk if you have questions. If you haven't been already, you can preview by opening `examples/index.html`.

File: dist/formrenderer.js [hunk 1]
```diff
@@ -1553,27 +1553,27 @@ window.JST["fields/address"] = function(__obj) {
       _print(_safe('\n\n'));
     
       if (format !== 'city_state' && format !== 'city_state_zip' && format !== 'country') {
-        _print(_safe('\n  <div class=\'fr_input_grid\'>\n    <div class=\'fr_item_full\'>\n      <input type="text"\n             id="'));
+        _print(_safe('\n  <div class=\'fr_input_grid\'>\n    <div class=\'fr_item_full\'>\n      <label class="fr_sub_label">Address</label>\n      <input type="text"\n             id="'));
```
Comment (jrubenoff) [inline dist/formrenderer.js]: Why would you keep the labels on the bottom if they appear on the top?

Comment (ajb) [inline dist/formrenderer.js]: Unless I misunderstand, the label _is_ on top. Let's comment on the non-minified diff, it might be clearer there.

**Comment by jrubenoff** [comment]
Made some CSS fixes, please review.

**Comment by ajb** [comment]
Am I crazy or did my previous comment disappear? here's what it was:

===

Are these CSS tweaks for formrenderer, or Screendoor? 

1. You'll notice that for better or worse, formrenderer uses `px` values in its CSS. I'd like to either stick with this, or rework the entire file to use `rem` or `em` units.

2. There is a layout error:

img

**Comment by jrubenoff** [comment]
1) I think this is OK in this instance. All labels are in relative units (`line-height: 1.5`), so `.label_sub` should as well.

`.fr_item_spacer` has a padding equal to the height of the label, and a `line-height` equal to the heights of the inputs, which are also set in relative units:

screen shot 2015-01-13 at 3 23 21 pm

2) Fixed.

**Comment by ajb** [comment]
Cool, that last change wasn't compiled FYI. (You can also run `grunt watch` to watch your files and compile when they're changed.)

And there is still this issue: 

img

**Comment by ajb** [comment]
Alright, fixed and rebased.

**Pull request merged**